ucsdscheduleme/Controllers/TestScheduleController.cs

```diff
@@ -0,0 +1,233 @@
+                };
+                Meeting CSE20Discussion1 = new Meeting()
+                {
+                    MeetingType = MeetingType.Discussion,
+                    Days = Days.Wednesday,
+                    StartTime = 1000,
+                    EndTime = 1050,
+                    Code = "A02"
+                };
+                Meeting CSE20Discussion2 = new Meeting()
+                {
+                    MeetingType = MeetingType.Discussion,
+                    Days = (Days.Tuesday | Days.Thursday),
+                    StartTime = 1711,
+                    EndTime = 1712,
+                    Code = "B02"
+                };
+                Course CSE20 = new Course()
+                {
+                    CourseAbbreviation = "CSE20",
+                    CourseName = "Intro / Discrete Mathematics",
+                    Units = 4,
+                    Description = ""
+                };
+                Section CSE20Section1 = new Section()
+                {
+                    Ticket = 201,
+                    Course = CSE20
+                };
+                Section CSE20Section2 = new Section()
+                {
+                    Ticket = 202,
+                    Course = CSE20
+                };
+                // Adding different sections to the course object
+                CSE20.Sections.Add(CSE20Section1);
+                CSE20Section1.Meetings.Add(CSE20Lecture);
+                CSE20Section1.Meetings.Add(CSE20Discussion1);
+                CSE20Section2.Meetings.Add(CSE20Lecture);
+                CSE20Section2.Meetings.Add(CSE20Discussion2);
+                CSE20.Sections.Add(CSE20Section2);
+
+                Meeting CSE21Lecture = new Meeting()
+                {
+                    MeetingType = MeetingType.Lecture,
+                    Days = (Days.Tuesday | Days.Thursday),
+                    StartTime = 1700,
+                    EndTime = 1710,
+                    Code = "A00"
+                };
+                Meeting CSE21Discussion1 = new Meeting()
+                {
+                    MeetingType = MeetingType.Discussion,
+                    Days = Days.Wednesday,
+                    StartTime = 900,
+                    EndTime = 901,
+                    Code = "A02"
+                };
+                Meeting CSE21Discussion2 = new Meeting()
+                {
+                    MeetingType = MeetingType.Discussion,
+                    Days = Days.Wednesday,
+                    StartTime = 903,
+                    EndTime = 904,
+                    Code = "A02"
+                }; 
+                Course CSE21 = new Course()
+                {
+                    CourseAbbreviation = "CSE21",
+                    CourseName = "Intro / Discrete Mathematics",
+                    Units = 4,
+                    Description = ""
+                };
+
+                Section CSE21Section1 = new Section()
+                {
+                    Ticket = 211,
```

Comment:
why do we need ticket here?

Reply:
i think it uniquely identifies the class with just a number